pkg/resource/aws/classic_loadbalancer.go

```diff
@@ -0,0 +1,3 @@
+package aws
```

Comment:
I would have called this file just like the Terraform resource type `aws_elb.go` (be aware of the AWS prefix) and change the test file as well. 

But the naming is good `AwsClassicLoadBalancerResourceType`